client/client.go

```diff
@@ -91,22 +91,22 @@ type doer interface {
 
 // Config allows the user to customize client behavior.
 type Config struct {
-	// BaseURL contains the base URL where the pebble daemon is expected to be.
-	// It can be empty for a default behavior of talking over a unix socket.
+	// BaseURL contains the base URL where the Pebble daemon is expected to be.
+	// It can be empty for a default behavior of talking over a Unix socket.
 	BaseURL string
 
-	// Socket is the path to the unix socket to use
+	// Socket is the path to the Unix socket to use.
```

Comment:
This should probably be kept lowercased, as we are not talking about the Actual Unix, but rather a unix-typed socket, so more of an adjective than the product name. People often call it *nix, but that seems over the top. Lowercasing should be enough.

Reply:
Fair -- updated similar instances to match.